### PR TITLE
added test for Tmin Tmax in kinetics data

### PR DESCRIPTION
**Pull request by davidfarinajr**
### Motivation or Problem
Kinetics data should have Tmin and Tmax, but the tests do not check for it. See https://github.com/ReactionMechanismGenerator/RMG-database/issues/573

### Description of Changes
Added a test to check for Tmin and Tmax in kinetics entries.

### CODE REVIEW

File: testing/databaseTest.py [hunk 1]
```diff
@@ -196,6 +202,12 @@ def test_kinetics(self):
             self.compat_func_name = test_name
             yield test, library_name
 
+            test = lambda x: self.kinetics_check_have_Tmin_Tmax(library)
+            test_name = "Kinetics depository {0}: check Tmin and Tmax?".format(library_name)
```
Comment (xiaoruiDong) [inline testing/databaseTest.py:206]: Minor: Replace `Kinetics depository` with `Kinetics library`?

**Review by xiaoruiDong — COMMENTED** [review]
@davidfarinajr Thanks for adding the test! Looks good! I only have a minor comment. Please see if it makes sense. Feel free to squash the fixes and rebase. I will approve and merge the PR. Cheers!

**Comment by davidfarinajr** [comment]
Looking at the tests, a lot of training depositories and kinetics libraries are missing Tmin and Tmax (https://github.com/ReactionMechanismGenerator/RMG-Py/runs/5726638557?check_suite_focus=true).  Should we only test the training depository?  Is this where it becomes an issue, when adding rules from training?

**Comment by xiaoruiDong** [comment]
@davidfarinajr We discussed forcing family training kinetics and library kinetics to have Tmin and Tmax recorded in the last RMG subgroup (Mar 30 2022), and we plan to do it during the next Hackathon (or one more if we cannot finish). I will merge this PR in after making sure Tmin and Tmax are added.

**Pull request closed without merging**